### ISSUE TITLE
FCOS3D assign

### ISSUE THREAD

**Issue by excitohe** [open-mmlab/mmdetection3d#1282]
Hello! If 
According to [LINE:881](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/fcos_mono3d_head.py#L881) and [LINE:890](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/fcos_mono3d_head.py#L890)
`gt_bboxes_3d = gt_bboxes_3d[None].expand(num_points, num_gts, self.bbox_code_size)`
`bbox_targets_3d = torch.cat((delta_xs, delta_ys, depths, gt_bboxes_3d[..., 3:]), dim=-1)`
So shape of **bbox_targets_3d** and **gt_bboxes_3d** are both [num_points, num_gts, self.bbox_code_size]
Then after:
https://github.com/open-mmlab/mmdetection3d/blob/32a4328b16b85aae26d08d81157ab74b58edcdb1/mmdet3d/models/dense_heads/fcos_mono3d_head.py#L935
we get **dists** with shape of  [num_points, num_gts]
https://github.com/open-mmlab/mmdetection3d/blob/32a4328b16b85aae26d08d81157ab74b58edcdb1/mmdet3d/models/dense_heads/fcos_mono3d_head.py#L938
However, `dists.min(dim=1)` get return value **min_dist** with shape of [num_points] and **min_dist_inds** with shape of [num_points].
Why not `dists.min(dim=0)`? Thx!
